playbooks/uda-workflow/connector-install/install_sap_hana_plugin.yml

```diff
@@ -0,0 +1,68 @@
+# => Cohesity Agent Management
+# =>
+# => Collection: cohesity.dataprotect
+# =>
+
+# => Install the Cohesity Datastore Agent on each SapHana host
+# => specified in the Ansible inventory
+# =>
+---
+- hosts: saphana
+  # => We need to gather facts to determine the OS type of
+  # => the machine
+  gather_facts: true
+  vars:
+    state: "present"
+    platform: SapHana
+    saphana_admin: sssadm
+    saphana_password: fr8shst8rt
+    saphana_home: /usr/sap/SSS/home
+    SID: SSS
+    user_store_key: SYS_KEY
+  collections:
+    - cohesity.dataprotect
+  tasks:
+    - name: Check if firewall is enabled on CentOS or RedHat
+      command: "firewall-cmd --state"
+      ignore_errors: true
+      register: firewall_status
+      when:
+        - ansible_distribution == "CentOS" or ansible_distribution == "RedHat" or ansible_distribution == "SLES"
+      tags: always
+
+    - name: Enable tcp port 11113 for CentOS or RedHat
+      command: "firewall-cmd {{ item }}"
+      with_items:
+        - --zone=public --permanent --add-port 11113/tcp
+        - --reload
+      when:
+        - ansible_distribution == "CentOS" or ansible_distribution == "RedHat" or ansible_distribution == "SLES"
+        - firewall_status.rc == 0
+      tags: always
+
+    - name: "Cohesity agent: Set Agent to state of {{ state | default('present') }}"
+      cohesity_plugin:
+        cluster: "{{ cohesity_server }}"
+        username: "{{ cohesity_username }}"
+        password: "{{ cohesity_password }}"
+        validate_certs: "{{ cohesity_validate_certs }}"
+        state: "{{ state | default('present') }}"
+        platform: "SapHana"
+        endpoint: "{{ inventory_hostname }}"
+        download_location: "{{ saphana_home }}"
+        netmask_bits: 20
+      register: result
+      when: state == "present" and platform == "SapHana"
```

Comment:
Make sure user understands state == present ==> install and state == absent ==> uninstall